src/components/services/AccountCallbackManager.tsx

```diff
@@ -104,9 +104,14 @@ export function AccountCallbackManager(props: Props) {
       }),
 
       wallet.on('wcNewContractCall', obj => {
-        const { dApp, payload, uri, walletId } = obj
-        if (walletId == null) return
-        Airship.show(bridge => <WcSmartContractModal bridge={bridge} wallet={wallet} dApp={dApp} payload={payload} uri={uri} />)
+        try {
+          const clean = asWcSmartContractModalProps(obj)
+          const { dApp, payload, uri } = clean
+
+          Airship.show(bridge => <WcSmartContractModal bridge={bridge} wallet={wallet} dApp={dApp} payload={payload} uri={uri} />)
+        } catch (e: any) {
+          console.warn('Invalid wcNewContractCall params', e.message)
```

Comment:
Don't do `e.message`. You can do `console.warn(..., e)` or even `console.warn('... ${String(e)}')`